plugins/modules/inventory_intent.py

```diff
@@ -263,6 +256,25 @@
         description: Location of the sites allocated for the APs
         type: list
         elements: str
+      provision_wired_device:
+        description: A list of dictionaries containing the IP address of wired devices and the site name where they will be provisioned.
+        type: list
+        elements: dict
+        suboptions:
+          device_ip:
+            description: The IP address of the wired device.
+            type: str
+          site_name:
+            description: The complete name of the site where the wired device will be provisioned(For example, "Global/USA/San Francisco/BGL_18/floor_pnp").
```

Comment:
description: Indicates the exact location where the wired device will be provisioned. This is a string value that should represent the complete hierarchical path of the site (For example, "Global/USA/San Francisco/BGL_18/floor_pnp").

Reply:
updated.